draft-ietf-rtcweb-jsep.xml

```diff
@@ -1595,7 +1595,27 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               "max-bundle", and this is not the first m= section, or the BUNDLE
               policy is set to "balanced", and this is not the first m= section
               for this media type, an "a=bundle-only" line.</t>
-            </list></t>
+
+	      <t>If the RtpTransceiver's RtpSender is active, an "a=rid" line
```

Comment:
I'd rather just wait until the transceiver PR has landed to submit this one.  The other way around would be more work.